src/Faker/Provider/da_DK/Payment.php

```diff
@@ -16,4 +16,83 @@ public static function bankAccountNumber($prefix = '', $countryCode = 'DK', $len
     {
         return static::iban($countryCode, $prefix, $length);
     }
+
+    /**
+     * Returns an account number with between 7 and 10 digits
+     *
+     * @link https://amcbanking.com/kb/10540/
+     * @return string
+     */
+    public static function localBankAccountNumber()
+    {
+        return self::numerify('###%######');
+    }
+
+    /**
+     * Returns an bank identifier/registration number with four digits
+     *
+     * @link https://amcbanking.com/kb/10540/
+     * @return string
+     */
+    public static function localBankRegistrationNumber()
+    {
+        return self::numerify('####');
+    }
+
+    /**
+     * @see static::localBankRegistrationNumber()
+     * @return string
+     */
+    public static function localBankIdentifier()
+    {
+        return static::localBankRegistrationNumber();
+    }
+
+    /**
+     * Sources:
+     * A list of all banks in Denmark
+     * @see https://skiftbank.dk/banker-i-danmark/
```

Comment:
Well, this source is copyrighted, we can only accept datasets coming from public domain or MIT compatible sources